### DISCUSSION TITLE
But why no update?

### DISCUSSION THREAD

**Discussion by Arelius-D** [qdm12/ddns-updater#723] [General]
Hey, just a general question I guess.

So, shouldn't DDNS send IP and update at the provider? _In my case it is [noip.com](https://www.noip.com/)_

If so then I guess it's a matter of bad configuration from my side or..? _I'm asking since I don't see any updates since I manually updated at the providers homepage._

![image](https://github.com/qdm12/ddns-updater/assets/168124713/247648fd-1022-4d47-b311-cd75ca015f80)

**Comment by qdm12** [comment]
If your public IP address is the same as the IP address stored in your A and/or AAAA record, then no update is made. I guess this is your case right? You can test this by force-setting your DNS A record to `127.0.0.1`, and the program should then send an update request after a few minutes.

**Comment by Arelius-D** [comment]
> If your public IP address is the same as the IP address stored in your A and/or AAAA record, then no update is made. I guess this is your case right? You can test this by force-setting your DNS A record to `127.0.0.1`, and the program should then send an update request after a few minutes.

It did and so it works as intended! Something else was also at play which I was not aware of, the fact that NoIP requires monthly log-in, so I ditched them and went over to DuckDNS instead.
